functions/image/New-DcnImage.ps1

```diff
@@ -616,6 +619,45 @@
                 }
             }
 
+
```

Comment:
This is a little bit of duplication, could we not make a base $params and then selective add in File or Query depending on the type?